lib/google_contacts_api/contact.rb

```diff
@@ -1,19 +1,36 @@
 module GoogleContactsApi
+
   # Represents a single contact.
-  # Methods we could implement:
-  # :categories, (:content again), :links, (:title again), :email
-  # :extended_properties, :deleted, :im, :name,
-  # :organizations, :phone_numbers, :structured_postal_addresses, :where
   class Contact < GoogleContactsApi::Result
-    # Returns the array of links, as link is an array for Hashie.
-    def links
-      self["link"].map { |l| l.href }
+
+    attr_reader :json, :original_contact
```

Comment:
You should put these on two separate lines, with comments for each describing their use case.